clases/person.rb

```diff
@@ -1,15 +1,16 @@
 require_relative 'corrector'
 
 class Person
-  attr_reader :id
   attr_accessor :name, :age
+  attr_reader :id, :parent_permission, :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rental = []
   end
 
```

Comment:
- [ ] Kindly add a method for making rentals, this will achieve the `has-many` relationship of a person having many rentals